pkg/kubectl/cmd/auth/cani.go

```diff
@@ -393,3 +399,22 @@ func isNamespaced(gvr schema.GroupVersionResource, discoveryClient discovery.Dis
 
 	return false, fmt.Errorf("the server doesn't have a resource type '%s' in group '%s'", gvr.Resource, gvr.Group)
 }
+
+func isVerb(o *CanIOptions) bool {
+	nonResourceURLVerbs := []string{"get", "put", " post", "head", "options", "delete", " patch"}
+	resourceVerbs := []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection", "use", "bind", "impersonate", "*"}
+
```

Comment:
can construct package variable `resourceVerbs := sets.NewString(...)` and do `resourceVerbs.Has(verb)`